chain/store/snapshot.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/filecoin-project/lotus/chain/types"
 )
 
+const MAX_TIPSETS_FROM_SNAPSHOT = 2000
```

Comment:
Not sure if this is the best place to put this or if we already have this constant available somewhere. Happy to take suggestions

Reply:
I would define this as 2 finalities instead of hardcoding -- I think that's a reasonable description of "minimum meaningful snapshot".

And I would rename it something specific to the TSK calculation (eg. TIPSETKEY_BACKFILL_RANGE).

And if you felt inspired, you could make it controllable by an envvar, but I honestly don't think there'll be much use for that functionality (can always do later).

Reply:
What do you mean by define as "2 finalities"? Are you referring to a constant for a finality defined as number of tipsets?

Reply:
done